src/path.ml

```diff
@@ -8,35 +8,34 @@ let of_string s = s
 
 let to_string s = s
 
-let is_absolute t = Filename.is_absolute t
+let delimiter = Node.Path.delimiter.[0]
```

Comment:
Do we need to assume it's a single character? Can't we just split on a string?

Reply:
Otherwise, let's add `assert (String.length delimiter = 1)`.

Reply:
[path.delimiter](https://nodejs.org/api/path.html#path_path_delimiter) will always return `:` or `;` as a JS string and `String.split` expects a `char`.